src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="month month--sun month--31">
```

Comment:
Still have problems with changing start day. You should specify modifiers and work with them


**Requirements:**
_Implement start-day modifier for the calendar with mon, tue, wed, thu, fri, sat and sun values
Use [@each](https://sass-lang.com/documentation/at-rules/control/each) to create all the modifiers
The month should start at the correct column (Monday is the 1st, Friday is the 5th)
You can just add correct margin-left for the first day
Set calendar to start from Sunday by default_

_Add a modifier month-length for the calendar with values 28, 29, 30 and 31 (use @for)
It sets the last day to show (use [nth-child](https://css-tricks.com/how-nth-child-works/))
Set 31 days by default_

```suggestion
      <div class="month month--start-day--sun month--month-length--31">
```

If you need any help feel free to ask at fe_chat